src/components/create-safe/steps/OwnerPolicyStep.tsx

```diff
@@ -30,7 +30,7 @@ const OwnerPolicyStep = ({ params, onSubmit, setStep, onBack }: Props): ReactEle
     address: wallet?.address || '',
   }
 
-  const defaultThreshold = params.threshold || 1
+  const defaultThreshold = 1
```

Comment:
We need to use `params.threshol` in order to preserve the value between steps. Try going forward then back a step and you'll see it remains as 1.

Reply:
I saw that no `params.threshold` were being kept between steps. Perhaps it's a different bug to fix. I will revert this line then

Reply:
reverted